cli/smartcontract/smart_contract.go

```diff
@@ -69,6 +69,10 @@ var (
 	}
 )
 
+// ModVersion contains `pkg/intepop` module version
```

Comment:
Typo in comment: s/`pkg/intepop`/`pkg/interop`.

Reply:
Fixed.